recipes-qt/automotive/qtapplicationmanager-noop.inc

```diff
@@ -3,4 +3,9 @@
 #   SPDX-License-Identifier: MIT
 #
 
-# this file is intentionally left blank
+# this file is intentionally left blank: it is sourced by the _append recipe for
```

Comment:
Use of ":" is wrong here AFAICT. LIke @jonte, i don't like this hack either FWIW.

Reply:
Please elaborate on the ":". I think it's correct to use a colon to clarify the previous statement, but I'm open to learn something new :-)

As for the hack, I fully agree with you, but I couldn't come up with a better alternative. I've googled for a day and this was the best I could find :-)

Reply:
It would be correct if the phrase before ':' would suggest that something is to follow. E.g "The reason for leaving this file blank is: ".

Oh and please please always capitalize the first letter in text that's supposed to be any European language. :)

Reply:
About the hack, I guess if @jonte wasn't able to suggest a better solution, there is not likely a better way. :)

Reply:
I've fixe the capitalization on the other PR. As for the colon, I believe it's not wrong, given how it's used in this page: https://www.grammarbook.com/punctuation/colons.asp (search for "judgement call" in the text)

Reply:
that's for the rule "A colon instead of a semicolon may be used between independent clauses when the second sentence explains, illustrates, paraphrases, or expands on the first sentence." where as in here it's more like "phrase: reason" instead of "phrase : explanation/expansion of phrase". Anyway, getting really nitpicky now. :D No biggie anyway.